main/inc/lib/sessionmanager.lib.php

```diff
@@ -5290,8 +5290,11 @@ public static function getAllUsersFromCoursesFromAllSessionFromStatus(
                 true,
                 false
             );
-            $studentIdList = array_keys($studentListSql);
-            $studentListSql = "'".implode("','", $studentIdList)."'";
+            if(!empty($studentListSql)){
```

Comment:
Here too (missing space after if). I'm proceeding with these changes directly, so don't worry for this time.